snippets/write-json-to-file.md

```diff
@@ -0,0 +1,9 @@
+### Write a JSON to a file
+
+Write a `json` object to a `.json` file.
+```js
+const fs = require('fs')
+
```

Comment:
I like this snippet. I can improve the description, but the code is solid. Remove the other file and I will merge this one!

Reply:
ok I will make things ok ASAP